MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2023, Enoque Macanda
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
```

Comment:
- [ ] Please delete these two lines, Please also remember everything in your MIT and README files should be about you and your project. Everything should be customized.

Reply:
This was my worst review process experience so far.